frontend/packages/core/src/Feedback/tooltip.tsx

```diff
@@ -37,7 +37,7 @@ export interface TooltipProps extends Pick<MuiTooltipProps, "interactive" | "pla
 
 const Tooltip = ({ children, maxWidth = "300px", title, ...props }: TooltipProps) => {
   return (
-    <StyledTooltip title={title} maxWidth={maxWidth} {...props}>
+    <StyledTooltip title={title} maxwidth={maxWidth} {...props}>
```

Comment:
thanks for fixing!

Reply:
Yeah I thought it was something I was doing initially lol